Core/Core/Configuration/Configurable/ConfigMock.swift

```diff
@@ -78,7 +78,19 @@ public class ConfigMock: Configurable {
 
     // MARK: - Configuratable -
 
-    public var socialLoginEnable: Bool {
+    public var isAppleSignInEnabled: Bool {
+        false
+    }
+
+    public var isGoogleSignInEnabled: Bool {
+        false
+    }
+
+    public var isFacebookSignInEnabled: Bool {
+        false
+    }
+
+    public var isMicrosoftSignInEnabled: Bool {
```

Comment:
@eyatsenkoperpetio How are you planning to use Facebook and other authentication provider keys in the configuration? Are you planning to directly add them to info.plist?

Reply:
we should to discuss about it, can we configure authentication providers without plist like Google?

Reply:
we should extend this configuration file

Reply:
I'm afraid we can't do that with all the authentication providers, for example, Facebook, Microsoft needs some keys to be added to the info.plist

We are achieving this by reading the values from the config file at compile time through a script and  adding them to the info.plist

Our config was look like this 
```
FACEBOOK:
    ENABLED: true
    FACEBOOK_APP_ID: 'facefoob_app_id'
    CLIENT_TOKEN: 'facefoob_client_token'
```